planner/dispatcher.go

```diff
@@ -0,0 +1,51 @@
+package planner
+
+import (
+	"context"
+	"github.com/weihesdlegend/Vacation-planner/iowrappers"
+	"sync"
+)
+
+const NumWorkers = 10
+
```

Comment:
why 10, ? and where do the workers run ?

Reply:
10 is just a tentative number. We can increase this or put it in configurations if needed.

Reply:
nice.. is this related to scalability in the sense , tune based on how fast or slow the results are returned back based on the number of `workers` we have.